### ISSUE TITLE
A11y violation on Search control: The WAI-ARIA role 'search' is not valid for the element <input>

### ISSUE THREAD

**Issue by schuetza** [carbon-design-system/carbon-components-vue#1237]
## Detailed description
When checking our app with the IBM A11y checker plugin, we found this violation
`The WAI-ARIA role 'search' is not valid for the element <input>`

After removing the search role from the  CvSearch onput field, the violation is gone.

![image](https://user-images.githubusercontent.com/68960267/128822031-372ccaf6-4582-48ba-99f7-d4efc353b4b8.png)

> Describe in detail the issue you're having.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

yes, CvSearch

> What did you expect to happen? What happened instead? What would you like to see changed?

Component shouldn't have A11y violations

> What browser are you working in?

Firefox 90

> What version of the Carbon Design System are you using?

carbon-vue

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

no time pressure

## Steps to reproduce the issue

1. Open storybook https://vue.carbondesignsystem.com/?path=/story/components-cvsearch--default
2. Open IBM A11y checker and scan the page
3. check the violation in the checker
4. remove the role from the input field (the outer div has the role, too)
5. rescan -> violation is gone

## Additional information

- Screenshots or code
- Notes
